modules/videopress/class.videopress-player.php

```diff
@@ -108,8 +108,8 @@ public function __construct( $guid, $maxwidth = 0, $options = array() ) {
 
 			if ( ! defined( 'WP_DEBUG' ) || WP_DEBUG !== true ) {
 				$expire = 3600;
-				if ( isset( $video->expires ) && is_int( $video->expires ) ) {
-					$expires_diff = time() - $video->expires;
+				if ( isset( $this->video->expires ) && is_int( $this->video->expires ) ) {
```

Comment:
With `unset( $video );` above, it looks like `isset( $video->expires )` will never be true

Reply:
Nice catch.